modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -284,31 +201,86 @@ Runtime Fabric is not deployed behind a load balancer, these values should not b
 
 |=== 
 
-== Internal Load Balancer Logs
-
+. (Optional) Configure Internal Load Balancer Logs
++
 You can define the log levels for the internal load balancer. Runtime Fabric supports the following log levels, listed in descending order of verbosity:
++
+** FATAL
+** ERROR
+** WARNING
+** INFO
+** VERBOSE
+** DEBUG
+** TRACE
++
+The more verbose log levels, which include WARNING, INFO, VERBOSE, DEBUG, and TRACE, consume more CPU resources for each request. Consider this when adjusting the log level and allocating resources for the internal load balancer.
++
+By default, the activity across all IPs addresses behind your endpoint is logged. To help reduce CPU consumption when using more verbose log levels, add IP filters to only log-specific IP addresses. This feature also reduces the quantity of logs when debugging a connection for a specific or limited number of IP addresses.
+
```

Comment:
GLOBALLY search for "select" and reevaluate for use of "click," per our style reference.